NotificationSystem/SendModeratorEmail.cs

```diff
@@ -57,13 +59,16 @@ public static async Task Run(
             // TODO: make better email
             string body = EmailTemplate.GetModeratorEmailBody(documentTimeStamp, location);
 
-            log.LogInformation("Retrieving email list and sending notifications");
+			var timeConstraint = TimeLimiter.GetFromMaxCountByInterval(14, TimeSpan.FromSeconds(1));
+			var aws = new AmazonSimpleEmailServiceClient(RegionEndpoint.USWest2);
+			log.LogInformation("Retrieving email list and sending notifications");
             foreach (var emailEntity in EmailHelpers.GetEmailEntities(cloudTable, "Moderator"))
             {
-                string emailSubject = string.Format("OrcaHello Candidate at location {0}", location);
+				await timeConstraint;
```

Comment:
What is the expected behavior? We send individual emails to each moderator/subscriber per detection. It looks like this is still keeping the same number of total emails, but only sending 14/sec.

Reply:
Yes it will limit email send 14/sec to avoid throttling. AWS SES throttles email if it exceed 14/sec.